docker-compose.yml

```diff
@@ -1,3 +1,318 @@
-# Stub file to get paths in all docker-compose files to work from project root.
-# Need this per: https://github.com/docker/compose/issues/3874
-version: '3.7'
+services:
+  startup:
+    extends:
+      file: common.yml
+      service: oada-uservice
+    image: oada/startup:${OADA_VERSION-build}
+    build:
+      args:
+        OADA_SERVICE: startup
+    container_name: startup
+    depends_on:
+      - zookeeper
+      - kafka
+      - arangodb
```

Comment:
It's been on my todo list to better understand "depends_on".  It always seems to work the opposite of what I think it does.  This *seems* like it would mean that restarting kafka will restart startup because startup depends on kafka, but what it actually does is when you restart startup, it restarts kafka.  That seems backwards to me, but want to investigate further.  It's unrelated to this PR because it was in there originally anyway, but this just reminded me of it.